smithy-protocol-test-traits/src/main/resources/META-INF/smithy/smithy.test.smithy

```diff
@@ -285,3 +285,119 @@ string NonEmptyString
     },
 ])
 string AppliesTo
+
+/// Define how a malformed HTTP request is rejected by a server given a specific protocol
+@trait(selector: "operation")
+@length(min: 1)
+list httpMalformedRequestTests {
+    member: HttpMalformedRequestTestCase
+}
+
+@private
+structure HttpMalformedRequestTestCase {
+    /// The identifier of the test case. This identifier can be used by
+    /// protocol test implementations to filter out unsupported test
+    /// cases by ID, to generate test case names, etc. The provided `id`
+    /// MUST match Smithy's `identifier` ABNF. No two `httpMalformedRequestTests`
+    /// test cases can share the same ID.
+    @required
+    @pattern("[A-Za-z_][A-Za-z0-9_]+")
```

Comment:
You need to anchor this

Reply:
That's what I get for copy and pasting HttpRequestTestCase

Reply:
Hah! We need to anchor that one too!

Reply:
I added it as another commit to this PR.